test/integration/taxon_browsing_test.rb

```diff
@@ -90,6 +90,13 @@ class TaxonBrowsingTest < ActionDispatch::IntegrationTest
     then_page_has_meta_robots
   end
 
+  it 'does not show an email signup link on a non-live page' do
+    given_there_is_a_taxon_with_grandchildren
+    and_the_taxon_is_not_live
+    when_i_visit_the_taxon_page
+    then_i_cannot_see_an_email_signup_link
```

Comment:
Would it be better to add this assertion to the test above it (as well as changing the test name) as the set up for both tests are the same?